bundles/org.openhab.binding.kostalinverter/src/main/resources/OH-INF/thing/PIKO1020.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="kostalinverter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="kostalinverterpiko1020">
```

Comment:
The binding name don't need to be included in the Thing Type ID.
```suggestion
	<thing-type id="piko1020">
```

Reply:
See my comment from 7 Oct. "The binding name don't need to be included in the Thing Type ID." Is this the parameter you meant, preventing the Thing from being showed up in the UI? Actually this naming shouldn't prevent it from showing up.